geomesa-tools/README.md

```diff
@@ -647,25 +647,63 @@ For more documentation on converter configuration, check out the [geomesa-conver
 
 #### Enabling S3 Ingest
 
-Hadoop ships with an implementation of an S3 Native filesystem denoted by the prefix ``s3n://`` that can be enabled in
-the Hadoop configuration used with GeoMesa Tools. Edit the core-site.xml file in your Hadoop installation. These
-instructions apply to Hadoop 2.5.0 and higher. Note that you must have the environmental variable
-``HADOOP_MAPRED_HOME`` set properly in your environment.
+Hadoop ships with an implementation of a S3 filesystems that can be enabled in the Hadoop configuration used with GeoMesa Tools. GeoMesa Tools can perform ingest using both the second-generation (`s3n`) and third-generation (`s3a`) filesystems. Edit the `$HADOOP_CONF_DIR/core-site.xml` file in your Hadoop installation, as shown below. These instructions apply to Hadoop 2.5.0 and higher. Note that you must have the environment variable ``HADOOP_MAPRED_HOME`` set properly in your environment. Some configurations can substitute ``HADOOP_PREFIX`` in the classpath values below.
+
+Warning: AWS credentials are valueable. They pay for services and control read and write protection for data. If you are running GeoMesa on AWS EC2 instances, it is recommended to use s3a. With s3a, you can omit the Access Key Id and Secret Access keys from `core-site.xml` and rely on IAM roles. 
+
+##### s3a
```

Comment:
@vpipkt can you also make the doc changes here: https://github.com/locationtech/geomesa/blob/master/docs/user/commandline_tools.rst